### PR TITLE
update zebrafish README

### PR DESCRIPTION
**Pull request by zoepiran**
1. Updated zebrafish README to match the simulation readme style
2. fixed TedSim README

### CODE REVIEW

File: analysis/hu_zebrafish_linnaeus/README.rst [hunk 1]
```diff
@@ -1,17 +1,21 @@
-Reproducing the zebrafish analysis
-==================================
-The analysis of the zebrafish data is a two-step process; the first step is the calculation of transition matrices using
-moslin, and the second is the downstream analysis of these transition matrices.
+zebrafish analysis
```
Comment (Marius1311) [inline analysis/hu_zebrafish_linnaeus/README.rst:1]: I would capitalize: "Zebrafish analysis"

File: analysis/hu_zebrafish_linnaeus/README.rst [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+===============
+
+The analysis of the zebrafish data is a two-step process:
+
+1. Calculation of transition matrices using moslin.
+2. Downstream analysis of the transition matrices.
 
-We provide `code <https://github.com/theislab/moslin/blob/analysis/simulations/analysis/hu_zebrafish_linnaeus/run_sbatch_hu_moslin.py>`_
-to obtain the couplings between hearts at consecutive time points on a SLURM based cluster scheduling system.
 The input data files can be found on figshare: `scRNA-seq <https://figshare.com/account/projects/163357/articles/22494529>`_
```
Comment (Marius1311) [inline analysis/hu_zebrafish_linnaeus/README.rst:9]: I'm not sure how stable these links are. The safest option would be to refer to the collection, which has a stable doi: https://doi.org/10.6084/m9.figshare.c.6533377.v1

File: analysis/hu_zebrafish_linnaeus/README.rst [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-Apart from the couplings generated in the previous step, the downstream analysis additionally uses data files for
-single-cell annotation, cell type colors, lineage information and sample timepoints found
-`here <https://figshare.com/account/projects/163357/articles/22502974>`_.
+File description
+----------------
+.. csv-table::
+    :header: "File", "What it does"
+    :delim: |
+
+    `run_hu_moslin.sh` | Helper script used by ``run_sbatch_hu_moslin.py``. Calls the relevant functions from `utils.py` to run the simulations and performance analysis.
```
Comment (Marius1311) [inline analysis/hu_zebrafish_linnaeus/README.rst:18]: In .rst, I think you have to use double "`" to get the desired effect, i.e. ``run_hu_moslin.sh``

File: analysis/hu_zebrafish_linnaeus/README.rst [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
+    `run_sbatch_hu_moslin.py` | Main SLURM script to calculate the couplings between the hearts at consecutive time points. Calling the command ``python3 run_sbatch_hu_moslin.py`` will instantiate sbatch calls to calculate all couplings. The output of each coupling is saved as a csv files under ``DATA_DIR/output/``.
+    `hu_moslin_fit_couplings.py` | Main function called by ``run_hu_moslin.sh``. ``fit_couplings_all()`` is used to compute the couplings between all hearts at consecutive time points for the given input arguments (`alpha`, `epsilon`, `beta`, and `tau_a`). The couplings are saved as ``.csv`` files under ``DATA_DIR/output/``.
```
Comment (Marius1311) [inline analysis/hu_zebrafish_linnaeus/README.rst:20]: I think I would spell out what DATA_DIR points to here, i.e. `data/hu_zabrafish_linnaeus'

File: analysis/simulations/pan_tedsim/README.rst [hunk 5]
```diff
@@ -12,5 +12,4 @@ File description
     `run.py` | Main script to run the moslin/`LineageOT <https://www.nature.com/articles/s41467-021-25133-1>`_ analysis.
     `utils_analysis.py` | Utilities for data visualization in `ZP_2023-04-02_tedsim_analysis.ipynb`.
     `utils_run.py` | Utilities for `run.py`.
-    `ZP_2023-04-02_tedsim_analysis.ipynb` | Visualize the results - TedSim data and performance. Uses a pre-computed TedSim simulation to visualize the initial state tree, simulated tree and gene expression.
-    Imports the grid search results and visualizes the cost as a function of method and stochastic silencing rate. 
+    `ZP_2023-04-02_tedsim_analysis.ipynb` | Visualize the results - TedSim data and performance. Uses a pre-computed TedSim simulation to visualize the initial state tree, simulated tree and gene expression. Imports the grid search results and visualizes the cost as a function of method and stochastic silencing rate.
```
Comment (Marius1311) [inline analysis/simulations/pan_tedsim/README.rst:15]: I think you need "``"

**Review by Marius1311 — CHANGES_REQUESTED** [review]
Looks good, just a few minor changes!

File: analysis/hu_zebrafish_linnaeus/README.rst [hunk 6]
```diff
@@ -1,17 +1,21 @@
...
-single-cell annotation, cell type colors, lineage information and sample timepoints found
-`here <https://figshare.com/account/projects/163357/articles/22502974>`_.
+File description
+----------------
+.. csv-table::
+    :header: "File", "What it does"
+    :delim: |
+
+    `run_hu_moslin.sh` | Helper script used by ``run_sbatch_hu_moslin.py``. Calls the relevant functions from `utils.py` to run the simulations and performance analysis.
+    `run_sbatch_hu_moslin.py` | Main SLURM script to calculate the couplings between the hearts at consecutive time points. Calling the command ``python3 run_sbatch_hu_moslin.py`` will instantiate sbatch calls to calculate all couplings. The output of each coupling is saved as a csv files under ``DATA_DIR/output/``.
```
Comment (Bastiaanspanjaard) [inline analysis/hu_zebrafish_linnaeus/README.rst:19]:
```suggestion
    `run_sbatch_hu_moslin.py` | Main SLURM script to calculate the couplings between the hearts at consecutive time points. Calling the command ``python3 run_sbatch_hu_moslin.py`` will instantiate sbatch calls to calculate all couplings. The output of each coupling is saved as a ``.csv`` file under ``DATA_DIR/output/``.
```

**Review by Bastiaanspanjaard — CHANGES_REQUESTED** [review]
Just one comment for writing consistency.

File: analysis/simulations/pan_tedsim/README.rst (hunk 5 above)
Comment (zoepiran) [reply analysis/simulations/pan_tedsim/README.rst:15]: i prefer files not marked like @michalk8 had it for other readmes.

File: analysis/hu_zebrafish_linnaeus/README.rst (hunk 3 above)
Comment (zoepiran) [reply analysis/hu_zebrafish_linnaeus/README.rst:18]: i prefer files not marked as @michalk8 did for other readmes hence kept this format.

Comment (michalk8) [reply analysis/hu_zebrafish_linnaeus/README.rst]: Yes, it doesn't make a difference in the table (it cannot render the \`).

Comment (Marius1311) [reply analysis/hu_zebrafish_linnaeus/README.rst]: But it can render the ``, see my readme.

**Comment by zoepiran** [comment]
@Bastiaanspanjaard @Marius1311  need you to approve the PR :)

**Review by Bastiaanspanjaard — APPROVED** [review]

**Review by Marius1311 — APPROVED** [review]
thanks!

**Pull request merged**